src/hooks/usePersistentGlobalState.ts

```diff
@@ -27,3 +32,32 @@ export const createPersistentGlobalStateHook = <T extends object>(
 
     return () => [container.useGlobalState(key)[0], setGlobalState];
 };
+
+export const usePersistentGlobalState = <T extends object>(
```

Comment:
What's the difference between `usePersistentGlobalState` and the existing `createPersistentGlobalStateHook`?

Reply:
I'm experimenting with using SWR for global state instead of `react-hooks-global-state` that was last updated a year ago.

Reply:
Right, maybe this could be moved to a later PR as well then.